depends/packages/qt.mk

```diff
@@ -159,114 +151,132 @@ ifneq ($(LTO),)
 $(package)_config_opts_linux += -ltcg
 endif
 
-ifneq (,$(findstring clang,$($(package)_cxx)))
-  ifneq (,$(findstring -stdlib=libc++,$($(package)_cxx)))
-    $(package)_config_opts_linux += -platform linux-clang-libc++ -xplatform linux-clang-libc++
-  else
-    $(package)_config_opts_linux += -platform linux-clang -xplatform linux-clang
-  endif
-else
-  $(package)_config_opts_linux += -platform linux-g++ -xplatform bitcoin-linux-g++
-endif
-
-$(package)_config_opts_mingw32 = -no-opengl
+$(package)_config_opts_mingw32 = -no-pkg-config
+$(package)_config_opts_mingw32 += -no-opengl
 $(package)_config_opts_mingw32 += -no-dbus
 $(package)_config_opts_mingw32 += -no-freetype
-$(package)_config_opts_mingw32 += -xplatform win32-g++
-$(package)_config_opts_mingw32 += "QMAKE_CFLAGS = '$($(package)_cflags) $($(package)_cppflags)'"
-$(package)_config_opts_mingw32 += "QMAKE_CXX = '$($(package)_cxx)'"
-$(package)_config_opts_mingw32 += "QMAKE_CXXFLAGS = '$($(package)_cxxflags) $($(package)_cppflags)'"
-$(package)_config_opts_mingw32 += "QMAKE_LINK = '$($(package)_cxx)'"
-$(package)_config_opts_mingw32 += "QMAKE_LFLAGS = '$($(package)_ldflags)'"
-$(package)_config_opts_mingw32 += "QMAKE_LIB = '$($(package)_ar) rc'"
-$(package)_config_opts_mingw32 += -device-option CROSS_COMPILE="$(host)-"
-$(package)_config_opts_mingw32 += -pch
 ifneq ($(LTO),)
 $(package)_config_opts_mingw32 += -ltcg
 endif
+
+# CMake build options.
+$(package)_config_env := CC="$$($(package)_cc)"
+$(package)_config_env += CXX="$$($(package)_cxx)"
+$(package)_config_env_darwin := OBJC="$$($(package)_cc)"
+$(package)_config_env_darwin += OBJCXX="$$($(package)_cxx)"
+
+$(package)_cmake_opts := -DCMAKE_PREFIX_PATH=$(host_prefix)
+$(package)_cmake_opts += -DQT_FEATURE_cxx20=ON
+$(package)_cmake_opts += -DQT_ENABLE_CXX_EXTENSIONS=OFF
+$(package)_cmake_opts += --log-level=STATUS
```

Comment:
In 428942fdd64b477eef809c2b2fb0906472f2449c:
Shouldn't this be `NOTICE` unless `V=1` or `DEBUG=1`; otherwise, why are we defaulting to the most verbose logging?